hack/build/target.sh

```diff
@@ -1,6 +1,17 @@
 #!/bin/sh
 set -e
 
+if [ -e "/etc/alpine-release" ]
```

Comment:
can you change this to

```
if [ -z "${TARGET_LIBC}" ] && [ -e "/etc/alpine-release" ] && [ "${KRATA_TARGET_IGNORE_LIBC}" != "1" ]
```

This script is in need of a cleanup, but this is how you can ignore libc specification in build scripts that need that.